reactors/1D_packbed.ipynb

```diff
@@ -0,0 +1,519 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #56.        for k in range(n_surf):
</pre><p>I don't think you need to do this in a loop, you can just assign directly to the array.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Cantera/cantera-jupyter/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Corrected.</p>